docs/data/sources/google-tag-manager-client.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Amplitude Analytics Browser SDK
+description: Official Google Tag Manager template for Amplitude Analytics for data collection.
```

Comment:
Please be more clear about which template this is for in the description to help disambiguate search results and create clear social previews.

Reply:
Added the name of the template and changes are as following:

```
Collect data with ease using Amplitude Analytics Browser SDK GTM template - the official client-side Google Tag Manager template for seamless data collection.
```